crates/bevy_ecs/src/observer/trigger_event.rs

```diff
@@ -14,13 +14,19 @@ pub struct TriggerEvent<E, Targets: TriggerTargets = ()> {
     pub targets: Targets,
 }
 
-impl<E: Event, Targets: TriggerTargets> Command for TriggerEvent<E, Targets> {
-    fn apply(mut self, world: &mut World) {
+impl<E: Event, Targets: TriggerTargets> TriggerEvent<E, Targets> {
+    pub(super) fn trigger(mut self, world: &mut World) {
```

Comment:
I'd personally leave this out of the PR since it introduces two ways of doing the same thing.